Testing/Dashboard/elxDashboardCommon.cmake

```diff
@@ -204,8 +200,8 @@ if( dashboard_model STREQUAL Continuous )
     endif()
 
     # Check for changes
-    ctest_update( SOURCE ${CTEST_DASHBOARD_ROOT} RETURN_VALUE res )
-    message( "Found ${res} changed files" )
+    # ctest_update( SOURCE ${CTEST_DASHBOARD_ROOT} RETURN_VALUE res )
+    # message( "Found ${res} changed files" )
```

Comment:
If these lines of code are no longer relevant, please just remove them. They will still be in git history anyway.

Reply:
Thanks for removing those lines!